src/state/playerSlice.ts

```diff
@@ -6,12 +6,14 @@ export interface PlayerState {
   name?: string;
   isHost: boolean;
   score: number;
+  score: number;
 }
 
 export const defaultPlayerState: PlayerState = {
   name: `The Champion (a.k.a. Joel)`,
```

Comment:
We can probably reset this to a blank string :P

Reply:
lol I liked the name